src/components/TooltipNew/TooltipPortal.js

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as PropTypes from 'prop-types';
+
+class TooltipPortal extends React.Component {
```

Comment:
it's copy + paste from ModalPortal component. keep it DRY please

Reply:
I used ModalPortal then...